examples/wifi-echo/server/esp32/main/ember-print.cpp

```diff
@@ -0,0 +1,68 @@
+#include <stddef.h>
+#include <stdio.h>
+#include <string.h>
+
+#include <support/CHIPLogging.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+bool emberAfPrintReceivedMessages = true;
+
+using namespace chip::Logging;
+void emberAfPrint(int category, const char * format, ...)
+{
+    if (format != NULL)
```

Comment:
I think this is an unnecessary check

Reply:
LGTM, I'm good for NULL checks 👍

Reply:
I hate to crash the accessory just cause a log statement had a null format string accidentally. 
IMHO crashes / exceptions are only when progressing any further means severe damage e.g. irrecoverable data loss.
Also it is super cheap to test this.

Reply:
is log() ever called anywhere with a non-const format string?  those would be coding errors?

Reply:
Coding errors happen everywhere :) NULL checks save lives.